docs/scipp-neutron/groupby.ipynb

```diff
@@ -186,7 +186,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The result has dimension `spectrum` replaced by the physically meaningful `scattering_angle` and the resulting plot is easily interpretable:"
+    "The result has dimension `spectrum` replaced by the physically meaningful `scattering_angle` and results in the same plot as before with histogrammed data."
```

Comment:
Not sure what the correction relates to. Don't we want to say that this is easily interpretable (or more scientifically useful)?

Reply:
I agree that the old version is what we want here.

Reply:
Sure, but it was just copy-pasted from the previous section. I just tried to make it a little more interesting to read.

Reply:
I see. I don't have a strong opinion then.